fuel/app/classes/materia/widget/instance.php

```diff
@@ -387,6 +388,35 @@ public function db_remove()
 		return true;
 	}
 
+	/**
+	 * 'Undeletes' instance by updating is_deleted flag from 1 to 0
+	 * @return bool true if successfully undeleted, false if unable to restore
+	 */
+	public function db_undelete()
```

Comment:
add return type hint

```suggestion
	public function db_undelete(): bool
```